api/api.yaml

```diff
@@ -180,6 +180,7 @@ definitions:
         description: Array of counts for each facet value.
       time_names:
         type: array
+        # This is string because time may be "Unknown"
```

Comment:
In the API I think it's technically "unknown" (not too important).

Reply:
Ah. In general the API server should pass what to render, UI server shouldn't have to make changes. How about we store in Elasticsearch as "Unknown"?

Reply:
Sounds good, I'll make that change.